src/clr-angular/progress/progress-bars/_variables.progress-bars.scss

```diff
@@ -5,6 +5,7 @@
 // Usage: ../progress/progress-bars/_progress-bars.clarity.scss
 $clr-progress-defaultBarColor: $clr-color-action-600 !default;
 // @deprecated in 2.0
+//  warning, success, and danger distinctions for progress bars are deprecated in 3.0
 $clr-progress-success-color: $clr-color-success-400 !default;
 // @deprecated in 2.0
```

Comment:
should the comments be updated to just say all of them will be deprecated in 3.0 (that is, remove the lines `// @deprecated in 2.0`)? otherwise I might think we missed removal of deprecated items when we review for 3.0.

Reply:
Yep. I'll do that. Thanks.